tests/controls/windowtest.cpp

```diff
@@ -144,11 +194,36 @@ TEST_CASE_METHOD(WindowTestCase, "Window::FocusEvent", "[window]")
 
 TEST_CASE_METHOD(WindowTestCase, "Window::Mouse", "[window]")
 {
+    wxInitAllImageHandlers();
     wxCursor cursor(wxCURSOR_CHAR);
     m_window->SetCursor(cursor);
 
     CHECK(m_window->GetCursor().IsOk());
 
+#if defined( __WXMSW__ ) || defined( __WXOSX__ )
+    wxCursor resCursor1( "horse" );
+    CHECK( resCursor1.IsOk() );
+#endif
+#ifdef __WXGTK__
+    // According to documentation cursors on X/GTK are different than in Windows/OSX.
+    // On X/GTK, using the constructor with wxString parameter is loading the cursor
+    // from the external file and not creating it from embedded bits/resources
+    // Therefore different path are used for creating embedded cursors
+    wxCursor resCursor1( (const char *) horse, 32, 32, 16, 0, nullptr, wxWHITE, wxBLACK );
+    CHECK( resCursor1.IsOk() );
+#endif
+#ifdef __WXOSX__
+    wxRemoveFile( wxStandardPaths::Get().GetResourcesDir() + "/horse.png" );
+    // Check the cur file
+    wxCursor resCursor3( "horse" );
+    CHECK( resCursor3.IsOk() );
+    wxRemoveFile( wxStandardPaths::Get().GetResourcesDir() + "/horse.cur" );
+    // Check that loading fails if the cursor file is not found
+    wxCursor resCursor2( "horse" );
+    CHECK( resCursor2.IsOk() );
+    wxCopyFile( "horse.png", wxStandardPaths::Get().GetResourcesDir() + "/horse.png" );
```

Comment:
This won't work unless the horse images are in the current working directory?
Should also not be needed if you follow above suggestion.

Reply:
@MaartenBent ,
I tested and it worked fine.
Images are restored in the Bundle.